data/moves.js

```diff
@@ -19157,12 +19157,12 @@ let BattleMovedex = {
 		basePower: 0,
 		category: "Status",
 		desc: "Lowers the target's Speed by 1 stage. Until the target switches out, it takes 2x damage from Fire moves.",
-		shortDesc: "-1 Spe. Target takes 2x damage from Fire moves.",
+		shortDesc: "Lowers the target's Speed by 1 stage. Target takes 2x damage from Fire moves.",
```

Comment:
This is too long, it cuts off. 
![image](https://user-images.githubusercontent.com/11671268/69058348-7d23a100-09e1-11ea-8572-f55db96b852d.png)

Reply:
Maybe "-1 target Spe" ?

Reply:
![Opera Instantané_2019-11-18_151644_localhost psim us](https://user-images.githubusercontent.com/39522426/69059811-8b3fe480-0a16-11ea-94a1-c29e6b45e96e.png)
Maybe that's ok 
I don't know which verb will be better [takes, gets, receives (too long)], because it doesn't sound english for me.
I have another wording who's good too
![Opera Instantané_2019-11-18_152437_localhost psim us](https://user-images.githubusercontent.com/39522426/69060506-be36a800-0a17-11ea-9d5a-5df7138e6335.png)

Reply:
"Target gets -1 Spe and takes 2x damage from Fire."